.gitignore

```diff
@@ -2,6 +2,7 @@
 ansible-azure.cache
 *.retry
 vault_password_file
+ansible/roles/
```

Comment:
I don't think this line is necessary, by default (without `-p` or different `roles_path` config) `ansible-galaxy install` saves required roles into .ansible home folder, outside this repo. I even prefer letting this path tracked when we may start another role from scratch, without publishing it into other git repo or galaxy... What you think? 😃

Reply:
Nice tips @OctavioBR :clap: 

I'll change and test this approach.

Reply:
@OctavioBR  Changed. 

Thanks dude.